website/docs/proxy/index.md

```diff
@@ -6,12 +6,365 @@ tags: [podman-desktop, installing, windows, macos, linux]
 keywords: [podman desktop, containers, podman, installing, installation, windows, macos, linux]
 ---
 
-# Using Podman Desktop behind a proxy
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-You can use Podman Desktop behind a proxy in various situations, such as:
+# Installing and using Podman Desktop and Podman in a restricted environment
 
-- [Using a proxy on Windows and macOS](/docs/proxy/using-a-proxy)
-- [Using a proxy requiring a custom Certificate Authorities on macOS and Windows](/docs/proxy/using-a-proxy-requiring-a-custom-ca)
-- [Using a proxy on Linux](/docs/proxy/using-a-proxy-on-linux)
-- [Using a proxy in your containers](/docs/proxy/using-a-proxy-in-your-containers)
-- [Accessing resources behind a VPN with Podman on Windows](/docs/proxy/using-a-vpn-on-windows)
+In a restricted environment you might face this challenges:
+
+- The default Podman Desktop and Podman installation methods download assets during the setup.
+
+  However, a network restricted environment might refuse access to these external resources.
+
+  Consider rather using a specific installation method.
+
+- The Podman Machine receives a network address distinct to your computer network address.
+
+  However, resources in your VPN might be available only for the traffic going through your computer network address.
+
+  Consider enabling the **User mode networking** option when creating your Podman Machine to route the network traffic through your VPN.
+
+- The Podman Machine connects directly to the external network.
+
+  However, a restricted environment might block all traffic no going to a proxy.
+
+  Consider configuring Podman Desktop and your Podman Machine to route the traffic through a proxy.
```

Comment:
Weird formatting happening here? Lot's of one line sentences (markdown doesn't add a newline except for double line spacing between sentences).

May be better in a paragraph?